doc/lua_api.md

```diff
@@ -7464,6 +7464,14 @@ use the provided load and write functions for this.
 * `from_file(filename)`: Experimental. Like `from_string()`, but reads the data
   from a file.
 
+`AsyncJob`
+----------
+An `AsyncJob` is a reference to a job to be run in an async environment.
+
+### Methods
+* `cancel()`: try to cancel the job
```

Comment:
I think this should mention that currently, this can only cancel jobs which have not begun execution yet. (Side note: Modders could use IPC to poll whether a job should die during execution.)

Reply:
Done. Although I'm not sure whether this should exactly mention using IPC for job cancellation.